Kernel/Devices/Audio/AC97/AC97.cpp

```diff
@@ -35,6 +35,15 @@ UNMAP_AFTER_INIT ErrorOr<NonnullRefPtr<AudioController>> AC97::create(PCI::Devic
 UNMAP_AFTER_INIT ErrorOr<bool> AC97::probe(PCI::DeviceIdentifier const& device_identifier)
 {
     VERIFY(device_identifier.class_code().value() == to_underlying(PCI::ClassID::Multimedia));
+
+    // If BAR0 is 64-bit then BAR1 isn't usable
```

Comment:
Why is that?

Reply:
Bar registers are 32-bit. So 64-bit bars use 2 registers consuming both bar0 and bar1. Added this to the comment

Reply:
Can you point me to a doc about this? I can't find any reference to BAR1 being unavailable in 64-bit addressing mode, but that might be a google skill issue on my end :man_shrugging:

Reply:
https://wiki.osdev.org/PCI#Address_and_size_of_the_BAR . There you see that 64-bit bar uses not just bar[x] but also bar[x+1]

Reply:
OK, so if we combine that with section 2.1 from the spec:
https://www.intel.com.tw/content/dam/doc/manual/io-controller-hub-7-hd-audio-ac97-manual.pdf

We know that the BAR addresses should always be 32-bit.